src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -104,6 +106,9 @@ const ArtworkAuctionCreateAlertHeader: FC<ArtworkAuctionCreateAlertHeaderProps>
     artwork?.savedSearch?.suggestedArtworksConnection?.totalCount ?? 0
   const displaySuggestedArtworksSection = suggestedArtworksCount > 0
   const displaySeeMoreButton = suggestedArtworksCount > 5
+  const isBidder = artwork.myLotStandingManageAlerts?.[0]
+  const isHighest = artwork.myLotStandingManageAlerts?.[0]?.isHighestBidder
+  const isUnderbidder = isBidder && !isHighest && !artwork.isInAuction
```

Comment:
**Suggestion:** How about we move this logic to MP along with `isHighestBidder` and have it therefore available in Eigen later as well and allow quick adjustments?

Reply:
To make sure I am getting this right, the logic is:
1. you have a bid for the active lot
2. You are not the highest bidder
3. this lot is not in an auction anymore
then you are an under bidder.
Why is condition 3 necessary? it doesn't seem trivial to me

Reply:
Ideally yes - we would have a specific field for the underbidder in MP, if there is room for that, let's do it!
The underbidder is someone who lost to the highest bidder, so as you said: closed lot/not in auction and not the highest bidder.

Reply:
Feel free to merge, my changes are not required anymore. A rename would be enough to solve my misunderstanding.

Reply:
I'm guessing the `isInAuction` guard is necessary in case the auction this artwork is in is still ongoing, but is there a guard for that higher up somewhere? I'd guess this entire component isn't even rendered unless the auction is already closed, which makes that check here not necessary.

Reply:
@mzikherman there was a misleading error for a failing test but I've  now removed `!isInAuction`.